pkgs/servers/home-assistant/default.nix

```diff
@@ -302,6 +305,8 @@ in python.pkgs.buildPythonApplication rec {
       getPackages
       python
       supportedComponentsWithTests;
+    customComponents = lib.recurseIntoAttrs (callPackage ./custom-components {});
+    lovelaceModules = lib.recurseIntoAttrs (callPackage ./lovelace-modules {});
```

Comment:
@piegamesde Your idea was to create `home-assistantPackages` instead, can you elaborate how that would benefit us?

Reply:
Just so I understand, is the intention here to ensure that dependencies of custom components are resolved via the home assistant package overrides?

Reply:
That's unrelated to this discussion, but yes, putting them into `extraPackages` was the plan.

Reply:
I don't remember how serious the "home-assistantPackages" proposal was. Maybe, but only a little? ^^ Anyways, I really like the package set idea even though it is breaking.